src/cli/subcommand/install.rs

```diff
@@ -139,6 +140,20 @@ impl CommandExecute for Install {
             }
         }
 
+        println!(
+            "\
+            {success}\n\
+            To use Nix, open a new terminal or run `{shell_reminder}`\n\
```

Comment:
```suggestion
            To get started using Nix, you may need to start a new shell, open a new terminal, or run `{shell_reminder}`\n\
```

Maybe something like this?

Reply:
Ya I was trying to keep it short so it didn't take multiple lines. I can do this instead though (though I will remove that extra comma you suggested.)

Reply:
Not my Oxford comma 😭

Reply:
That's not an oxford comma though! It's not a list.

Reply:
"You may need to" kinda doesn't make sense here since they definitely need to...

Reply:
I think we're talking about different commas lol -- I didn't think you were talking about the first comma (`Nix, [...]`), but the last comma (`, or [...]`)😆 

Yeah, seems fine :)